courses/machine_learning/deepdive2/text_classification/labs/classify_text_with_bert.ipynb

```diff
@@ -0,0 +1,1190 @@
+{
```

Comment:
<p>Nit: Maybe consider breaking up this print block and mixing it with a one sentence explanation of each? E.g. I was unclear what the type ID was on first pass from the output thinking something was wrong before reading 4 lines down what it referred to.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1412/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>